diff --git a/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Amf/AmfStitchVideoSampler.cpp b/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Amf/AmfStitchVideoSampler.cpp
index e01bd563b5..2716ba764a 100644
--- a/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Amf/AmfStitchVideoSampler.cpp
+++ b/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Amf/AmfStitchVideoSampler.cpp
@@ -55,11 +55,11 @@
 /* FAmStitchVideoSampler structors
 *****************************************************************************/
 
-FAmfStitchVideoSampler::FAmfStitchVideoSampler(FAmfStitchMediaTracks& InTracks, amf::AMFContextPtr InAmfContext, bool bUseDifferentGPUAdapter)
-    : Tracks(InTracks)
+FAmfStitchVideoSampler::FAmfStitchVideoSampler(FAmfStitchMediaTracks& InTracks, amf::AMFContextPtr InAmfContext, bool bUseDifferentGPUAdapter):
+	UseDifferentGPUAdapter(bUseDifferentGPUAdapter)
+	, Tracks(InTracks)
     , BufferList(7) // MaxCapacityPlusOne
     , AmfContext(InAmfContext)
-    , UseDifferentGPUAdapter(bUseDifferentGPUAdapter)
 {
     AmfDevice = static_cast<ID3D11Device*>(AmfContext->GetDX11Device());
 }
diff --git a/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaPlayer.cpp b/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaPlayer.cpp
index 9d53922e02..c73392b0a6 100644
--- a/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaPlayer.cpp
+++ b/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaPlayer.cpp
@@ -58,9 +58,9 @@
 
 FAmfStitchMediaPlayer::FAmfStitchMediaPlayer(IMediaEventSink& InEventSink)
     : EventSink(InEventSink)
-    , MediaUrl(FString())
-    , Info(FString())
-    , bLooping(false)
+	, Info(FString())
+	, MediaUrl(FString())
+	, bLooping(false)
     , Tracks(new FAmfStitchMediaTracks)
 {
 }
diff --git a/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaTextureSample.h b/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaTextureSample.h
index 95c36631f6..4a27dd31aa 100644
--- a/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaTextureSample.h
+++ b/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaTextureSample.h
@@ -50,10 +50,10 @@ public:
     /** Default constructor. */
     FAmfStitchMediaTextureSample()
         : SampleFormat(EMediaTextureSampleFormat::Undefined)
-        , Time(FTimespan::Zero())
-        , Duration(FTimespan::Zero())
+		, bIsCubemap(false)
+		, Duration(FTimespan::Zero())
+		, Time(FTimespan::Zero())
         , Texture(nullptr)
-        , bIsCubemap(false)
     {}
 
     /** Virtual destructor. */
diff --git a/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaTracks.cpp b/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaTracks.cpp
index ca0fdd0d9b..3877167987 100644
--- a/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaTracks.cpp
+++ b/Engine/Plugins/Media/AmfStitchMedia/Source/AmfStitchMedia/Private/Player/AmfStitchMediaTracks.cpp
@@ -48,9 +48,9 @@
 
 FAmfStitchMediaTracks::FAmfStitchMediaTracks()
     : VideoSamplePool(new FAmfStitchMediaTextureSamplePool)
-    , SelectedVideoTrack(INDEX_NONE)
     , SampleFormat(EMediaTextureSampleFormat::Undefined)
     , bUseCubemap(false)
+	, SelectedVideoTrack(INDEX_NONE)
 {
 }
 
diff --git a/Engine/Plugins/Runtime/Sphere360/Source/Sphere360/Private/Sphere360MeshComponent.cpp b/Engine/Plugins/Runtime/Sphere360/Source/Sphere360/Private/Sphere360MeshComponent.cpp
index 3650ae86c7..a0934d38af 100644
--- a/Engine/Plugins/Runtime/Sphere360/Source/Sphere360/Private/Sphere360MeshComponent.cpp
+++ b/Engine/Plugins/Runtime/Sphere360/Source/Sphere360/Private/Sphere360MeshComponent.cpp
@@ -298,7 +298,7 @@ public:
 		if (bWireframe)
 		{
 			WireframeMaterialInstance = new FColoredMaterialRenderProxy(
-				GEngine->WireframeMaterial ? GEngine->WireframeMaterial->GetRenderProxy(IsSelected()) : NULL,
+				GEngine->WireframeMaterial ? GEngine->WireframeMaterial->GetRenderProxy() : NULL,
 				FLinearColor(0, 0.5f, 1.f)
 			);
 
@@ -311,7 +311,7 @@ public:
 		{
 			if (Section != nullptr && Section->bSectionVisible)
 			{
-				FMaterialRenderProxy* MaterialProxy = bWireframe ? WireframeMaterialInstance : Section->Material->GetRenderProxy(IsSelected());
+				FMaterialRenderProxy* MaterialProxy = bWireframe ? WireframeMaterialInstance : Section->Material->GetRenderProxy(/*IsSelected()*/);
 
 				// For each view..
 				for (int32 ViewIndex = 0; ViewIndex < Views.Num(); ViewIndex++)
@@ -334,7 +334,31 @@ public:
 							Mesh.bWireframe = bWireframe;
 							Mesh.VertexFactory = &Section->VertexFactory;
 							Mesh.MaterialRenderProxy = MaterialProxy;
-							BatchElement.PrimitiveUniformBuffer = CreatePrimitiveUniformBufferImmediate(GetLocalToWorld(), GetBounds(), GetLocalBounds(), true, UseEditorDepthTest());
+
+							bool bHasPrecomputedVolumetricLightmap = false;
+							FMatrix PreviousLocalToWorld;
+							int32 SingleCaptureIndex = 0;
+							GetScene().GetPrimitiveUniformShaderParameters_RenderThread(
+								GetPrimitiveSceneInfo(),
+								bHasPrecomputedVolumetricLightmap,
+								PreviousLocalToWorld,
+								SingleCaptureIndex
+								);
+
+							FDynamicPrimitiveUniformBuffer& DynamicPrimitiveUniformBuffer =
+								Collector.AllocateOneFrameResource<FDynamicPrimitiveUniformBuffer>();
+
+							DynamicPrimitiveUniformBuffer.Set(
+								GetLocalToWorld(),
+								PreviousLocalToWorld,
+								GetBounds(),
+								GetLocalBounds(),
+								true,
+								bHasPrecomputedVolumetricLightmap,
+								UseEditorDepthTest()
+								);
+							BatchElement.PrimitiveUniformBufferResource = &DynamicPrimitiveUniformBuffer.UniformBuffer;
+
 							BatchElement.FirstIndex = 0;
 							BatchElement.NumPrimitives = Section->IndexBuffer.Indices.Num() / 3;
 							BatchElement.MinVertexIndex = 0;
diff --git a/Engine/Plugins/Runtime/Sphere360/Source/Sphere360/Sphere360.Build.cs b/Engine/Plugins/Runtime/Sphere360/Source/Sphere360/Sphere360.Build.cs
index a9f92e20a2..6b3d3e7abe 100644
--- a/Engine/Plugins/Runtime/Sphere360/Source/Sphere360/Sphere360.Build.cs
+++ b/Engine/Plugins/Runtime/Sphere360/Source/Sphere360/Sphere360.Build.cs
@@ -47,7 +47,6 @@ namespace UnrealBuildTool.Rules
 					"CoreUObject",
 					"Engine",
 					"RenderCore",
-					"ShaderCore",
 					"RHI",
 					"MediaAssets",
 				}
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-10e090a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-10e090a.wav44.h
index b6e5bd3e3f..6c88eca5ab 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-10e090a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-10e090a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTFm10e090aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-30e048a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-30e048a.wav44.h
index e5ed286329..ebf6edc065 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-30e048a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-30e048a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTFm30e048aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-30e138a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-30e138a.wav44.h
index 489619ddba..1cf1358524 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-30e138a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H-30e138a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTFm30e138aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e000a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e000a.wav44.h
index 3bbecbab49..d9a3eaafb3 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e000a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e000a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF0e000aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e020a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e020a.wav44.h
index 3988506b55..f10bbe3e35 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e020a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e020a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF0e020aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e160a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e160a.wav44.h
index f1185e7c13..53261b91cc 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e160a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H0e160a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF0e160aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H20e090a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H20e090a.wav44.h
index 8ece136506..989039aa54 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H20e090a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H20e090a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF20e090aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H40e045a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H40e045a.wav44.h
index ac90c41d9c..7952db78cd 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H40e045a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H40e045a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF40e045aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H40e135a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H40e135a.wav44.h
index d1e14a99db..37a7db1410 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H40e135a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H40e135a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF40e135aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H70e000a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H70e000a.wav44.h
index 271f609090..71deed1f9b 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H70e000a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H70e000a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF70e000aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H70e180a.wav44.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H70e180a.wav44.h
index c0978000d0..8872b01aeb 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H70e180a.wav44.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers44.1/H70e180a.wav44.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF70e180aS44[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-10e090a.wav48.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-10e090a.wav48.h
index f8657a50ee..e525c13779 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-10e090a.wav48.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-10e090a.wav48.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTFm10e090aS48[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-30e048a.wav48.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-30e048a.wav48.h
index 028782d139..3687079194 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-30e048a.wav48.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-30e048a.wav48.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTFm30e048aS48[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-30e138a.wav48.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-30e138a.wav48.h
index 06cb4429ad..2dc1839855 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-30e138a.wav48.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H-30e138a.wav48.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTFm30e138aS48[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H0e020a.wav48.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H0e020a.wav48.h
index 1fffe00817..389c7916ca 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H0e020a.wav48.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H0e020a.wav48.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF0e020aS48[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H0e160a.wav48.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H0e160a.wav48.h
index 08ae78cc74..7a6cb2737f 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H0e160a.wav48.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H0e160a.wav48.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF0e160aS48[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H20e090a.wav48.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H20e090a.wav48.h
index 53fa3b05bc..3df5d8cc1f 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H20e090a.wav48.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H20e090a.wav48.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF20e090aS48[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H40e045a.wav48.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H40e045a.wav48.h
index a3ed752f10..1afa5f3d12 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H40e045a.wav48.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H40e045a.wav48.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF40e045aS48[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H40e135a.wav48.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H40e135a.wav48.h
index b82698d78d..086d7f5123 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H40e135a.wav48.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H40e135a.wav48.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF40e135aS48[] = 
diff --git a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H70e000a.wav48.h b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H70e000a.wav48.h
index b9bc2fed66..771d62a160 100644
--- a/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H70e000a.wav48.h
+++ b/Engine/Source/ThirdParty/AMD/AMF_SDK/amf/public/src/components/AmbisonicRenderer/measuredHRTF/headers48/H70e000a.wav48.h
@@ -1,3 +1,23 @@
+//***** The data in this file is reformatted from original work of MIT Media Laboratory *****//
+//***** The original copyright notice is below:                                                 *****//
+
+// This is a makehrtf HRIR definition file.  It is used to define the layout
+// and source data to be processed into an OpenAL Soft compatible HRTF.
+//
+// This definition is used to transform the left ear HRIRs from the full set
+// of KEMAR HRIRs provided by Bill Gardner <billg@media.mit.edu> and Keith
+// Martin <kdm@media.mit.edu> of MIT Media Laboratory.
+//
+// The data full.tar.Z or full.zip is available from:
+//
+//  http://sound.media.mit.edu/resources/KEMAR.html
+//
+// It is copyright 1994 by MIT Media Laboratory, and provided free of charge
+// with no restrictions on use so long as the authors above are cited.
+//
+// This definition is used to generate the internal HRTF table used by OpenAL
+// Soft.
+
 #pragma once
 
 const amf_uint8 HRTF70e000aS48[] = 
diff --git a/README.md b/README.md
index 2eeb55ede6..182fb2779c 100644
--- a/README.md
+++ b/README.md
@@ -6,7 +6,7 @@
 * Windows &reg; 7 (SP1 with the [Platform Update](https://msdn.microsoft.com/en-us/library/windows/desktop/jj863687.aspx)), Windows&reg; 8.1, or Windows&reg; 10.
 * Visual Studio&reg; 2015.
 * AMD Radeon Software Adrenalin Edition 18.3.4 or newer.
-* Unreal Engine 4.20.1 or newer.
+* Unreal Engine 4.22 or newer.
 
 AmfStitchMedia is compatible with most recent Radeon GPUs starting with the Southern Islands family and APUs of the Kabini, Kaveri, Carrizo families and newer.
 
@@ -83,7 +83,7 @@ The *Sphere 360* runtime plugin is bundled with *AmfStitchMedia* and implements
 * Select a stereoscopic *Mode*.
 * Assign the created media texture material to the sphere.
 
-### Third-Party Software 
+### Third-Party Software
 * FFmpeg is distributed under the terms of the LGPLv2.1.
 
 ### Attribution
